src/index.js

```diff
@@ -17,6 +17,12 @@ const UNNAMED = [];
 
 const VOID_ELEMENTS = /^(area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)$/;
 
+const DASHED_ATTRS = /^(acceptC|httpE|(clip|color|fill|font|glyph|marker|stop|stroke|text|vert)[A-Z])/;
+const CAMEL_ATTRS = /^(isP|viewB)/;
+const COLON_ATTRS = /^(xlink|xml|xmlns)([A-Z])/;
+
+const CAPITAL_REGEXP = /([A-Z])/g;
+
 const UNSAFE_NAME = /[\s\n\\/='"\0<>]/;
```

Comment:
Can they cover all the attributes? `react-dom` has an entire file for `possibleStandardNames`:

https://github.com/facebook/react/blob/ca106a02d1648f4f0048b07c6b88f69aac175d3c/packages/react-dom/src/shared/possibleStandardNames.js

If they can, at least add a unit test case to cover them all.